scalableminds/webknossos
# Dont brush outside viewport

This PR disables drawing out of bounds of a viewport

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- open volumetracing
- draw with the brush around the boundaries 
- move around to check that drawing stopped when drawing out of bounds
- there should not be noticeable difference in performance compared to the previous drawing 

### Issues:
- fixes  #3128

------
- ~~~[ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~~
- ~~~[ ] Updated [documentation](../blob/master/docs) if applicable~~~
- ~~~[ ] Needs datastore update after deployment~~~
